lib/suggestion-list-element.js

```diff
@@ -328,7 +326,7 @@ module.exports = class SuggestionListElement {
     if (wordContainer && wordContainer.offsetLeft) {
       this.uiProps.marginLeft = -wordContainer.offsetLeft
     }
-    if (!this.uiProps.itemHeight) {
+    if (!this.uiProps.itemHeight && this.selectedLi) {
```

Comment:
@50Wliu could the fix for https://github.com/atom/autocomplete-plus/issues/830 literally just be this?

This also had the added bonus of making all the tests pass.

Reply:
Could be!  Though I haven't ever looked at the ac+ codebase so I don't know.  Should get rid of the exception at the very least.